style.css

```diff
@@ -25,26 +25,43 @@ html {
 }
 
 body {
+  /* layout */
   display: grid;
   place-items: center;
+  position: relative;
+  /* visual */
   font-family: "Manrope", sans-serif;
   font-weight: 800;
   color: var(--p-cyan);
   background-color: var(--n-blue-dark);
-position: relative;
 }
 
 main {
   background-color: var(--n-blue-gray-dark);
 }
 
+h1 {
+  font-family: inherit;
+  color: var(--p-green);
+  font-size: 0.75rem;
+  text-transform: uppercase;
+}
+
 #dice-icon {
   background-color: var(--p-green);
   width: 64px;
   height: 64px;
   border-radius: 50%;
   display: grid;
   place-items: center;
+  cursor: pointer;
+}
+
+/* Don't select images that are just website layout */
+#dice-icon,
+picture {
+  -webkit-user-select: none;
+  user-select: none;
```

Comment:
It also drives me nuts when I come across a thing like this (copying image not allowed). In this particular case though, image serves as a button and nothing else, so there is no need to indicate it can be copied. Let's keep Nikki's changes.